src/components/menu/menu.test.js

```diff
@@ -1399,5 +1448,23 @@ context("Testing Menu component", () => {
           expect(callback).to.have.been.calledOnce;
         });
     });
+
+    it("should have correct keyboard navigation order when children of submenu update", () => {
```

Comment:
Are we missing some assertions for this test? I assume we would want to check the correct element is being focused after each arrow press?

Reply:
It's not really necessary here imo, the full keyboard behaviour is tested by other cases, this test is just to show that when the children update the focusable items in state are updated so that the order is correct

Reply:
Yep good point, looks like I've missed it in a few places, likely due to copy pasting 👏